scripts/dev/upstream_merge/upstream_merge.sh

```diff
@@ -48,6 +56,18 @@ parse_args() {
 sanity_test_repo() {
    local LOCAL_BASE_BRANCH=$1
 
+   # Optionally, attempt to checkout a branch before the sanity check (in case not using default)
```

Comment:
Am I correct in understanding that this change is equivalent to just skipping `sanity_test_repo` step since the next step `update_local_base_branch` already does `git checkout $LOCAL_BASE_BRANCH `?

And if that's the case, why not just have this option skip the `sanity_test_repo` step? I think `git rev-parse --verify $LOCAL_BASE_BRANCH` ceases to be useful anymore after we do a checkout.

Reply:
I agree. That does make more sense. I'll adjust it